modules/Configurator/controller.php

```diff
@@ -45,6 +45,49 @@
 require_once('include/MVC/Controller/SugarController.php');
 class ConfiguratorController extends SugarController
 {
+
+    /**
+     * Go to the authentication view
+     */
+    function action_gcauthentication(){
```

Comment:
Use the functions visibility (`public`), use PSR-2 coding standard on php files.